.env.example

```
@@ -0,0 +1,18 @@
+# App
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+
+# Better Auth
+BETTER_AUTH_URL=http://localhost:3000
+BETTER_AUTH_SECRET=replace-with-a-random-string
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the BETTER_AUTH_URL key

(UnorderedKey)